boot/seal.go

```diff
@@ -219,7 +220,7 @@ func resealKeyToModeenv(rootdir string, model *asserts.Model, modeenv *Modeenv,
 	resealKeyParams := &secboot.ResealKeyParams{
 		ModelParams:          modelParams,
 		KeyFile:              filepath.Join(InitramfsEncryptionKeyDir, "ubuntu-data.sealed-key"),
-		TPMPolicyAuthKeyFile: filepath.Join(dirs.SnapFDEDirUnder(rootdir), "tpm-policy-auth-key"),
+		TPMPolicyAuthKeyFile: filepath.Join(dirs.SnapSaveDirUnder(rootdir), "device/fde/tpm-policy-auth-key"),
```

Comment:
I wonder if we need a SnapFDESaveDirUnder helper?

Reply:
that or a const with `"device/fde"`.